tests/app/views/test_marketplace.py

```diff
@@ -335,6 +335,17 @@ def test_dos_brief_has_at_least_one_section(self):
         assert_equal(contract_length_key[0], 'Contract length')
         assert_equal(contract_length_value[0], '4 weeks')
 
+    def test_dos_brief_has_question_and_answer_session_details_link(self):
```

Comment:
Dare I say we could have a couple more tests... maybe one where the questions deadline has closed and one where there aren't any session details - in each case testing that the section isn't there?